Source/SwiftLintFramework/Models/Version.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Version.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 27/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
```

Comment:
👍 for not reading from the bundle, but I think it'd be better if we kept this namespaced inside a `struct` so we can avoid using `String` everywhere. This would also make it easier to implement version pinning if we want.